src/sonic-host-services/scripts/hostcfgd

```diff
@@ -36,6 +37,11 @@ RADIUS_SERVER_TIMEOUT_DEFAULT = "5"
 RADIUS_SERVER_AUTH_TYPE_DEFAULT = "pap"
 RADIUS_PAM_AUTH_CONF_DIR = "/etc/pam_radius_auth.d/"
 
+# MISC Constants
+CFG_DB = "CONFIG_DB"
+PRIO_INIT = 10
```

Comment:
What does this constant mean? Priority of what? Suggest to rename with more descriptive name